src/s3fs.cpp

```diff
@@ -2748,6 +2749,7 @@ static void* s3fs_init(struct fuse_conn_info* conn)
   if(is_remove_cache && !FdManager::DeleteCacheDirectory()){
     DPRNINFO("Could not inilialize cache directory.");
   }
+  #endif
```

Comment:
For __APPLE__ case, does FdManager::DeleteCacheDirectory() occur something error?